docker/daemon/upload/deploy.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "net/http"
+
+import (
+	"io"
+	"log"
+	"os"
+)
+
+const (
+	BaseDir = "/var/naglfar/lib/"
+)
+
+func upload(w http.ResponseWriter, r *http.Request) {
+	fileName := r.Header.Get("fileName")
+	err := os.Remove(BaseDir + fileName)
+	if err != nil {
+		log.Println(err)
+	}
+	f, err := os.OpenFile(BaseDir+fileName, os.O_WRONLY|os.O_CREATE, 0600|0066)
+	if err != nil {
+		log.Println(err)
+	}
+	io.Copy(f, r.Body)
+	defer f.Close()
+	log.Println("upload successfully", fileName)
+}
+
+func main() {
+	http.HandleFunc("/upload", upload)
```

Comment:
How to finalize the tmpFS you upload?

Reply:
![image](https://user-images.githubusercontent.com/59600278/105654725-8ff50800-5ef9-11eb-852f-d91946b87606.png)
Every time, it will delete haproxy.cfg every time, and then write a new one, which will not accumulate.

Reply:
What about if two clusters with HAProxy are deployed? You mean the second one overrides the first one's configuration?